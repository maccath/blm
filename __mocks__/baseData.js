const baseData = {
  site: {
    siteMetadata: {
      author: "Site Author",
      description: "Site description",
      title: "Site Title",
    },
  },
  placeholderImage: {
    childImageSharp: {
      fluid: {
        src: "icon.png",
        srcSet: "icon.png 50w, icon.png 100w",
        sizes: "(max-width: 200px) 100vw, 200px",
        base64: "data:image/png;base64,iVBORw0KG",
        aspectRatio: 1,
      },
    },
  },
}

export default baseData
