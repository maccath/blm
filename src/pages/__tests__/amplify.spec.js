import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import "jest-styled-components"
import baseData from "../../../__mocks__/baseData"

import Amplify from "../amplify"

beforeEach(() => {
  useStaticQuery.mockImplementation(() => baseData)
})

describe("Amplify page", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Amplify />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
