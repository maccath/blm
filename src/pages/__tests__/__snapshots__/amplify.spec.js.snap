// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amplify page renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": 960,
      "padding": "0 1.0875rem 1.45rem",
    }
  }
>
  <main>
    <h1>
      Amplify
    </h1>
    <p>
      We can combine our voices to enact social change. 
    </p>
    <ul>
      <li>
        <a
          href="https://www.writetothem.com/"
        >
          Write to your MP
        </a>
         demanding the fair treatment
of our Black citizens, to publicly condemn the actions of Donald Trump and the
police in America, the development of a strategy to fix race inequality, and
the suspension of riot gear export (see

        <a
          href="https://www.caat.org.uk/resources/export-licences/dashboard?region=United+States+of+America&item=anti-riot%2Fballistic+shields,components+for+anti-riot%2Fballistic+shields,anti-riot+shields,tear+gas%2Friot+control+agents,components+for+anti-riot+guns,components+for+civil+riot+control+agent+protection+equipment,acoustic+devices+for+riot+control,anti-riot+guns,civil+riot+control+agent+protection+equipment,maintenance+equipment+for+anti-riot+guns,technology+for+anti-riot%2Fballistic+shields,technology+for+the+use+of+anti-riot+guns,anti-riot+helmets,civil+riot-control+agent+protection+equipment,portable+incapacitating+substance+riot+control+devices,components+for+acoustic+devices+for+riot+control,portable+anti-riot+devices,portable+riot+control+electric-shock+devices,Tear+Gas+Group,components+for+civil+riot-control+agent+protection+equipment,components+for+portable+anti-riot+devices,components+for+portable+riot+control+electric-shock+devices,equipment+for+the+use+of+anti-riot+guns%22"
        >
          CAAT
        </a>
        
for a list of riot gear export license holders).
      </li>
      <li>
        Answer 
        <a
          href="https://docs.google.com/forms/d/e/1FAIpQLSc5AzoKpB904HmXAXlzJxKAt41YJ0cQ1zRlhMNe7PyfqEdxWA/viewform"
        >
          The Impact of Omission
        </a>
        
survey to investigate the extent to which British Imperial history is explored
in the curriculum of compulsory  education in the U.K.
      </li>
      <li>
        Sign the petition to 
        <a
          href="https://www.change.org/p/uk-parliament-suspend-uk-export-of-tear-gas-rubber-bullets-and-riot-shields-to-usa"
        >
          suspend riot gear export to the USA
        </a>
      </li>
      <li>
        Sign the petition to demand 
        <a
          href="https://www.change.org/p/uk-government-bame-leadership-a-demand-for-a-covid-19-race-equality-strategy"
        >
          a COVID19 race equality strategy
        </a>
      </li>
      <li>
        Sign the petition to 
        <a
          href="https://www.change.org/p/department-of-education-battle-racism-by-updating-reading-lists-at-gcse"
        >
          battle racism by updating GCSE reading lists
        </a>
      </li>
      <li>
        Send your friends a link to this page - 
        <a
          href="https://blm.katy-ereira.co.uk"
        >
          https://blm.katy-ereira.co.uk
        </a>
      </li>
    </ul>
    <div
      className="mt-6 flex space-x-4"
    >
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/donate"
      >
        I have some spare cash.
      </a>
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/learn"
      >
        I care and want to learn more.
      </a>
    </div>
  </main>
  <footer
    className="css-1bxtzlt"
  >
    <p>
      Compiled by 
      <a
        href="http://katy-ereira.co.uk"
      >
        Katy Ereira
      </a>
       /
       
      <a
        href="http://twitter.com/maccath"
      >
        @maccath
      </a>
       | Fist icon by
       
      <a
        href="https://thenounproject.com/cesarreynoso562/"
      >
        Cesar Reynoso
      </a>
    </p>
  </footer>
</div>
`;
